packages/node_modules/@ciscospark/media-engine-webrtc/src/webrtc-helpers.js

```diff
@@ -45,12 +45,27 @@ export function getMediaFromSDP(kind, sdp, direction) {
     type: kind
   };
 
+  let media = filter(parse(sdp).media, query);
+
+  const mediaTypes = {};
+
   if (direction) {
```

Comment:
`if (direction && direction === 'sendrecv')`

Reply:
That changes the logic and would fall through to the `else` path even if `direction` was undefined.